src/Model/Indexing/IndexFieldTypes.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Model\Indexing;
+
+final class IndexFieldTypes
+{
+    // ISO 8601 date "2004-02-12T15:19:21+00:00"
+    public const DATEFORMAT = 'c';
```

Comment:
Consider using [ISO8601_EXPANDED ](https://www.php.net/manual/en/class.datetimeinterface.php#datetimeinterface.constants.iso8601-expanded) or [ATOM](https://www.php.net/manual/en/class.datetimeinterface.php#datetimeinterface.constants.atom) (both are easier to read and understand the meaning of than `c`):
```suggestion
    public const DATEFORMAT = \DateTimeInterface::ISO8601_EXPANDED;
```

Reply:
Fixed